src/components/Header/Header.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Hidden from '@material-ui/core/Hidden';
+import Menu from '@material-ui/icons/Menu';
+import Button from '../Button/Button';
+import headerStyle from '../../assets/styles/components/headerStyle';
+
+const Header = ({ ...props }) => {
+  const makeBrand = () => {
+    let name;
+    props.routes.map(prop => {
```

Comment:
Why are you using a map to iterate the array?

Reply:
I didn't realize it wasn't necessary to use a map. I already changed to .find